packages/icons/icons.yml

```diff
@@ -279,62 +279,115 @@
     - 32
 - name: ai-results
   friendly_name: ai-results
-  aliases: []
+  aliases: 
+  - ai
+  - results
+  - health
+  - watson
   sizes:
     - 32
 - name: ai-results--high
   friendly_name: ai-results--high
-  aliases: []
+  aliases: 
+  - ai
+  - results
+  - health
+  - watson
+  - high
   sizes:
     - 32
 - name: ai-results--low
   friendly_name: ai-results--low
-  aliases: []
+  aliases: 
+  - ai
+  - results
+  - health
+  - watson
+  - low
   sizes:
     - 32
 - name: ai-results--medium
   friendly_name: ai-results--medium
-  aliases: []
+  aliases: 
+  - ai
+  - results
+  - health
+  - watson
+  - medium
   sizes:
     - 32
 - name: ai-results--urgent
   friendly_name: ai-results--urgent
-  aliases: []
+ aliases: 
```

Comment:
For future reference, any misalignment of these columns will cause errors and the icons will be unable to be built. Please ensure no whitespace is added or removed